xcode/Subconscious/Shared/Components/Feed.swift

```diff
@@ -45,13 +116,48 @@ struct FeedView: View {
             .ignoresSafeArea(.keyboard, edges: .bottom)
             .zIndex(2)
         }
+        .background(Color.background)
+        /// Replay some app actions on feed store
+        .onReceive(
+            app.actions.compactMap(FeedAction.from),
+            perform: store.send
+        )
+        /// Replay select feed actions on app
+        .onReceive(
+            store.actions.compactMap(AppAction.from),
+            perform: app.send
+        )
+    }
+}
+
+extension FeedAction {
+    static func from(_ action: AppAction) -> Self? {
+        switch action {
+        case .succeedIndexOurSphere(_):
+            return .refreshAll
+        case .succeedIndexPeer(_):
+            return .refreshAll
+        case .requestFeedRoot:
+            return .requestFeedRoot
+        default:
+            return nil
+        }
     }
 }
 
+extension AppAction {
+    static func from(_ action: FeedAction) -> Self? {
+        switch action {
+        default:
+            return nil
+        }
```

Comment:
Q: nothing replayed to app yet?

Reply:
Yeah just a placeholder, I can drop it for now. I wasn't sure if I'd need it during development.